tests/drivers/adc/adc_api/testcase.yaml

```diff
@@ -7,6 +7,7 @@ tests:
   drivers.adc:
     depends_on: adc
     min_flash: 40
+    min_ram: 20
```

Comment:
This does not look right. The test itself does not require much RAM. Couldn't those problematic platforms be filtered out in some other way?

Reply:
I don't see the issue here. This is the usual way to filter out tests/samples based on memory footprint.

Here is the footprint report for a build on `nrf52840dk_nrf52840`
```
Memory region         Used Size  Region Size  %age Used
           FLASH:       86488 B         1 MB      8.25%
             RAM:       16624 B       256 KB      6.34%
        IDT_LIST:          0 GB        32 KB      0.00%

```
So RAM consumption for this test is above 16K while `nucleo_u031r8` available RAM is 12K.

Reply:
> Here is the footprint report for a build on `nrf52840dk_nrf52840`

Here's one for `nrf51dk/nrf51822`:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:       43280 B       256 KB     16.51%
             RAM:        8112 B        32 KB     24.76%
        IDT_LIST:          0 GB        32 KB      0.00%
```
It fits in 8 kB. And that's the point. It depends on the platform, not the test itself.

Reply:
Interesting. Fair enough, let's filter it in another way.